packages/openclaw/skills/prefactor-fixup/SKILL.md

```
@@ -0,0 +1,37 @@
+---
+
+Fix the Prefactor OpenClaw plugin path mismatch after installation.
+
+## Problem
+
+When installing via `openclaw plugins install @prefactor/openclaw`, OpenClaw incorrectly installs the plugin to `~/.openclaw/extensions/openclaw` instead of `~/.openclaw/extensions/prefactor`.
+
+## Detection
+
+Run `openclaw plugins list` and check output for the specific warning:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)